pyxs/helpers.py

```diff
@@ -125,6 +126,16 @@ def validate_path(path):
     return path
 
 
+def path_exists(path):
+    """Checks if a given path exists, i.e. is an actual Xenstore path.
+    :exc:`~pyxs.exceptions.InvalidPath` for details
+    """
+    try:
+        return Client().ls(path) != None
+    except PyXSError as e:
+        raise NonexistentPath('%s is not an existing Xenstore path' % path)
```

Comment:
I think `path` should be the only argument here, because the rest is easily inferred from the exception class name.